### PR TITLE
XW-240 trigger openMainPage method when 403 error on /main/edit

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

**Comment by vforge** [comment]
LGTM

File: front/scripts/main/routes/CuratedContentEditorRoute.ts [hunk 1]
```diff
@@ -205,5 +163,52 @@ App.CuratedContentEditorRoute = Em.Route.extend(
...
 			}
+		},
+
+		/**
+		 * Called when user clicks on custom back button or after data is saved
+		 * Does transition to the main page or sends a message through Ponto if available
+		 *
+		 * @param dataSaved
+		 */
+		openMainPage(dataSaved: boolean = false) {
```
Comment (hakubo) [inline front/scripts/main/routes/CuratedContentEditorRoute.ts]: can this have a different name than action
like
handleOpeningMainPage

**Comment by hakubo** [comment]
LGTM

**Comment by vforge** [comment]
LGTM

**Pull request merged**